assets/js/blocks/cart-checkout/mini-cart/edit.tsx

```diff
@@ -52,6 +55,10 @@ const MiniCartBlock = ( {
 		} ),
 	} );
 
+	const themeSlug = getSetting( 'themeSlug', '' );
```

Comment:
I don't have a strong opinion here but we can get the theme data using Gutenberg selector, like the following snippet. Both are working for me but wondering if we have a preferred way to get the core data? cc @Aljullu 

```
	const currentTheme = useSelect( ( select ) =>
		select( coreStore ).getCurrentTheme()
	);
```

Reply:
Note that `useSelect` dispatch a REST request, so the theme data isn't available right after loaded. This is a disadvantage compared to the current way.

Reply:
> Note that `useSelect` dispatch a REST request, so the theme data isn't available right after loaded. This is a disadvantage compared to the current way.

That's a good point! I think we can go now with the solution @gigitux implemented and refactor it if needed in the future. How does it sound? In #5195 we will need to make the link dynamic, so we might need to do further adjustments to the logic introduced in this PR.

Reply:
@Aljullu Sounds good to me!